packages/noco-docs/docs/020.getting-started/050.self-hosted/020.environment-variables.md

```diff
@@ -29,20 +29,20 @@ For production use cases, it is crucial to set all environment variables marked
+| `NC_S3_ENDPOINT` | No        | S3 endpoint for S3 storage plugin.                                                                                                                                                                                  | Defaults to `s3.<region>.amazonaws.com` |
+| `NC_S3_ACCESS_KEY` | No        | The AWS access key ID for the S3 storage plugin. Required if no IAM role access.                                                                                                                                    |                                         |
+| `NC_S3_ACCESS_SECRET` | No        | The AWS access secret associated with the S3 storage plugin. Required if no IAM role access.                                                                                                                                                       |                                         |
+| `NC_S3_FORCE_PATH_STYLE` | No        | Whether to force [path-style requests](https://docs.aws.amazon.com/AmazonS3/latest/userguide/VirtualHosting.html#path-style-access) for the S3 storage plugin.                                                      |                                         |
+| `NC_S3_ACL` | No        | The [ACL](https://docs.aws.amazon.com/AmazonS3/latest/userguide/acl-overview.html) for the objects in S3                                                                                                            |        | |
+| `NC_ATTACHMENT_FIELD_SIZE` | No        | Maximum file size allowed for [attachments](/fields/field-types/custom-types/attachment/) in bytes.                                                                                                                 | Defaults to `20971520` (20 MiB).        |
+| `NC_MAX_ATTACHMENTS_ALLOWED` | No        | Maximum number of attachments allowed per cell.                                                                                                                                                                     | Defaults to `10`.                       |
+| `NC_ATTACHMENT_RETENTION_DAYS` | No        | Number of days to retain attachment on storage after all references deleted. (Set 0 to keep forever)                                                                                                                | Defaults to `10`.                       |
+| `NC_SECURE_ATTACHMENTS` | No        | Enables access to attachments only through pre-signed URLs. Set to `true` to activate; all other values are treated as `false`. ⚠ Note: Enabling this will make existing links inaccessible.                        | Defaults to `false`.                    |
+| `NC_ATTACHMENT_EXPIRE_SECONDS` | No        | Time in seconds after which pre-signed URLs for attachments start to expire. The actual expiration will occur after this time plus an additional 10 minutes. Only applicable if `NC_SECURE_ATTACHMENTS` is enabled. | Defaults to `7200` (2 hours). |
```

Comment:
Please split up your changes into two commits that follow the commit message convention
something along the lines of
- style: ***
- feat: ***